modules/dhcp_common/isc/omapi_provider.rb

```diff
@@ -200,7 +200,9 @@ def bootServer server
         ns = ip2hex validate_ip(server)
       rescue
         begin
-          ns = ip2hex Resolv.new.getaddress(server)
+          resolver = Resolv.new
+          resolver.timeouts = 5
```

Comment:
I don't think this exists so I doubt it has an effect.

```
[5] pry(main)> resolver = Resolv.new
=> #<Resolv:0x0000555ba96bd5f0
 @resolvers=
  [#<Resolv::Hosts:0x0000555ba96bd578
    @filename="/etc/hosts",
    @initialized=nil,
    @mutex=#<Thread::Mutex:0x0000555ba96bd528>>,
   #<Resolv::DNS:0x0000555ba96bd4b0
    @config=
     #<Resolv::DNS::Config:0x0000555ba96bd2d0
      @config_info=nil,
      @initialized=nil,
      @mutex=#<Thread::Mutex:0x0000555ba96bd258>,
      @timeouts=nil>,
    @initialized=nil,
    @mutex=#<Thread::Mutex:0x0000555ba96bd488>>]>
[6] pry(main)> resolver.timeouts
NoMethodError: undefined method `timeouts' for #<Resolv:0x0000555ba96bd5f0>
from (pry):6:in `__pry__'
```

Reply:
Yeah because this is not Resolv::DNS but just Resolv generic class.